README.md

```diff
@@ -156,6 +156,9 @@ export DOCKER_CERT_PATH=/Users/jonathan.li/.docker/machine/machines/default
 export DOCKER_MACHINE_NAME=default
 ```
 
+
+You can rerun with eval $(docker-vpn-helper) to set your environment variables
```

Comment:
I think it might make more sense to:

1. insert this between the existing steps 3 and 4
2. update the existing step 4 to note that the explicit `export`s are not required if you re-ran the script with `eval $(./docker-vpn-helper)`